app/views/registration/NextTaxYear.scala.html

```diff
@@ -92,11 +89,6 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
                                         <input type="hidden" id='actives_@{index}_uid' name='actives[@index].uid' value='@activeField("uid").value.get'/>
                                         <input class="govuk-checkboxes__input" type="checkbox" id='checkbox-@activeField("uid").value.get' name='actives[@index].active' value="true" @if(activeField("uid").value.get == "47"){aria-describedby="actives[@index].hint"}/>
                                         <label class="govuk-label govuk-checkboxes__label" for="checkbox-@activeField("uid").value.get">@messages("BenefitInKind.label." + activeField("uid").value.get)</label>
-                                        @if(activeField("uid").value.get == "47") {
```

Comment:
That if clock has been removed, because upper if already controlling that with `!=` and not possible to come here with `==`. 

Upper if :
```
@if(activeField("uid").value.isDefined && activeField("uid").value.get != "47"
```